src/modules/navigator/rtl_params.c

```diff
@@ -110,10 +110,10 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
  * Fly straight to the return location or planned mission landing and land there or
  * use the planned mission to get to those points.
  *
- * @value 0 Return home via direct path
- * @value 1 Return to a planned mission landing, if available, via direct path, else return to home via direct path
- * @value 2 Return to a planned mission landing, if available, using the mission path, else return to home via the reverse mission path
- * @value 3 Return via direct path to closest destination: home, mission landing pattern or safe point
+ * @value 0 Return to closest safe point (home or rally point) via direct path.
+ * @value 1 Return to closest safe point other than home (mission landing pattern or rally point), via direct path. If no mission landing or rally points are defined return home via direct path.
+ * @value 2 Same as above (2nd option), but return via mission path (for return to mission landing), via the reverse mission path (home) or via direct path (rally point).
```

Comment:
Suggest:
@value 2 Same as @value 1, but return via mission path (for return to mission landing), via the reverse mission path (for return to home) or via direct path (for return to rally point).

Reply:
Well this is a little complicated, because what is the safe point for the mission case? The mission landing  pattern or the closest waypoint? Also in current implementation behaviour differs if you're on a mission already or just flying around.
My assumption is that if you're in a mission you will always want to use this path, but perhaps that is wrong?

I can help write this, but can you confirm the assumptions?
1. On a mission then fast forward to end or reverse to back, irrespective of rally points?
1. Not on a mission then go to closest of rally points or nearest waypoint. 
1. If on mission path do fast forward or reverse depending on whether mission landing 
1. If no rally points or mission defined, go to home via direct path. 
1. If I reverse mission path, currently I then fly to home after takeoff position. Should I go to a rally point if it is closer than home at this point?